protons/calibration.py

```diff
@@ -421,17 +370,17 @@ class Tyrosine(Lysine):
     """
     Amber constant-pH TYR residue state weights at given pH
     """
-    pKa = 9.6
+    pka = 9.6
 
 
 class Cysteine(Lysine):
     """
     Amber constant-pH CYS residue state weights at given pH
     """
-    pKa = 8.5
+    pka = 8.5
 
 
-class CalibrationSystem(object):
+class AmberCalibrationSystem(object):
```

Comment:
Do you plan to migrate some of the reusable code from `AmberCalibrationSystem` into a base class usable by the `OpenMM` and `Amber` calibration variants in a future PR?